### PR TITLE
(feat) zoo-021: create website header

### PR DESCRIPTION
**Pull request by NikaKlokava**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -10,6 +10,33 @@ body {
...
+.banner-section .web-header {
+  width: 100%;
+  background: black;
+  position: absolute; /* we need it, because otherwise it takes the place below image */
+  top: 0px;
+  backdrop-filter: blur(2px);
+}
+.banner-section .web-header > * {
+  display: inline-block;
+  margin: 14px 0px 14px 220px;
```
Comment (Dzmitry-Klokau) [inline index.css:23]: please apply it only for div, not for all elements
Why? because in the next lines use set margin: 0

File: index.css [hunk 2]
```diff
@@ -10,6 +10,33 @@ body {
...
+  color: #ffffff;
+}
+.banner-section .web-header > a {
+  margin: 0px;
+  padding-left: 58px;
+  vertical-align: 8px;
+  text-decoration: none;
+}
+.banner-section .web-header > p {
```
Comment (Dzmitry-Klokau) [inline index.css:32]: ```anner-section .web-header > ``` and ```anner-section .web-header > p``` almost same

Comment (Dzmitry-Klokau) [inline index.css:32]: anner-section .web-header > and anner-section .web-header > p almost same

**Review by Dzmitry-Klokau — CHANGES_REQUESTED** [review]

File: index.css (hunk 2 above)
Comment (NikaKlokava) [reply index.css:32]: No! I have in .web-header 4 tags <a> and one <div> with logo, and also i have only one tag <p>, so i need to use styles only for ONE <p>, not for all section.

**Pull request merged**